lib/WP_Auth0_LoginManager.php

```diff
@@ -241,27 +241,11 @@ public function redirect_login() {
     $data = json_decode( $response['body'] );
 
     if ( isset( $data->access_token ) || isset( $data->id_token ) ) {
+  
       // Get the user information
-
-      if ( !isset( $data->id_token ) ) {
-        $data->id_token = null;
-        $response = WP_Auth0_Api_Client::get_user_info( $domain, $data->access_token );
-      } else {
-        try {
-          // grab the user ID from the id_token to call get_user
-          $decodedToken = JWT::decode( $data->id_token, $this->a0_options->get_client_secret_as_key(), array(  $this->a0_options->get_client_signing_algorithm() ) );
-        } catch (Exception $e) {
-          WP_Auth0_ErrorManager::insert_auth0_error('redirect_login/decode', $e->getMessage());
-          throw new WP_Auth0_LoginFlowValidationException(__('Error: There was an issue decoding the token, please review the Auth0 Plugin Error Log.', 'wp-auth0'));
-        }
-
-        // validate that this JWT was made for us
-        if ( $this->a0_options->get( 'client_id' ) !== $decodedToken->aud ) {
-          throw new Exception( 'This token is not intended for us.' );
-        }
-
-        $response = WP_Auth0_Api_Client::get_user( $domain, $data->id_token, $decodedToken->sub );
-      }
+  
+      $data->id_token = null;
```

Comment:
why you remove the id_token but not the access_token?

Reply:
@lbalmaceda - Not sure I'm totally clear on your question but this was basically 'if you have an id token, use it, otherwise use the access token' and we want to get rid of the first condition. I'm forcing it to use the access_token and left the `$data->id_token = null;` because I don't know what the downstream effects are of keeping that id_token around. Maybe there is another check like this down the line? Seemed safer to leave this than remove it.